tools/build/AppImage/build-with-conda.sh

```diff
@@ -0,0 +1,209 @@
+#     (replace the Mambaforge path by yours and zsh by your shell name)
+#
+#     2) appimagetool from https://appimage.github.io/appimagetool/
+#     place it in your exec path so it can be found by this script
+#
+#     3) a gpg key to sign the package
+#
+#     4) verify or change the config values here below
+#
+#     5) If you are on Fedora, there are lines to uncomment below in the script
+#
+# When done, just run this script and you should get an appimage with the
+# corresponding sha hash in the current folder.
+#
+# To cleanup after build: Just delete the "AppDir" folder (not done automatically since
+# next builds can reuse the downloaded packages).
+
+# config
+
+# make sure you have a gpg key for this email
+gpg_key="yorik@freecad.org"
+# the FreeCAD version we're looking for
+target_version="0.21.0"
+# make sure target_python matches the one FreeCAD is built with! Check with
+# conda search "freecad=0.21.0" -c adrianinsaval/label/dev -c freecad -c conda-forge
+target_python="3.10"
+
+# end config
+
+# export MAMBA_NO_BANNER=1
+
+# building needed files
+mkdir -p AppDir
+cat > AppDir/AppRun <<EOF
+#!/bin/bash
+HERE="\$(dirname "\$(readlink -f "\${0}")")"
+export PREFIX=\${HERE}/usr
+export LD_LIBRARY_PATH=\${HERE}/usr/lib\${LD_LIBRARY_PATH:+':'}\$LD_LIBRARY_PATH
+export PYTHONHOME=\${HERE}/usr
+export PATH_TO_FREECAD_LIBDIR=\${HERE}/usr/lib
+# export QT_QPA_PLATFORM_PLUGIN_PATH=\${HERE}/usr/plugins
+# export QT_XKB_CONFIG_ROOT=\${HERE}/usr/lib
+export FONTCONFIG_FILE=/etc/fonts/fonts.conf
+export FONTCONFIG_PATH=/etc/fonts
+# export QTWEBENGINEPROCESS_PATH=\${HERE}/usr/libexec/QtWebEngineProcess
+
+# Show packages info if DEBUG env variable is set
+if [ "\$DEBUG" = 1 ]; then
+    cat \${HERE}/packages.txt
+fi
+
+# SSL
+# https://forum.freecadweb.org/viewtopic.php?f=4&t=34873&start=20#p327416
+export SSL_CERT_FILE=\$PREFIX/ssl/cacert.pem
+# https://github.com/FreeCAD/FreeCAD-AppImage/pull/20
+export GIT_SSL_CAINFO=\$HERE/usr/ssl/cacert.pem
+# https://github.com/FreeCAD/FreeCAD-Bundle/issues/92#issuecomment-1086829486
+export QTWEBENGINE_DISABLE_SANDBOX=1
+# Support for launching other applications (from /usr/bin)
+# https://github.com/FreeCAD/FreeCAD-AppImage/issues/30
+if [ ! -z "\$1" ] && [ -e "\$HERE/usr/bin/\$1" ] ; then
+    MAIN="\$HERE/usr/bin/\$1" ; shift
+else
+    MAIN="\$HERE/usr/bin/freecad"
+fi
+
+\${MAIN} "\$@"
+EOF
+chmod +x AppDir/AppRun
+
+# building package name
+target_date=$(date +"%Y-%m-%d")
+arch=$(uname -m)
+package_name="FreeCAD_${target_version}-${target_date}-conda-Linux-${arch}-py$(echo ${target_python} | sed 's/\.//g')"
+conda_env="AppDir/usr"
+
+# dependencies
+echo "\nCreating the environment"
+packages="freecad=${target_version} occt vtk python=${target_python} \
+blas=*=openblas numpy matplotlib-base scipy sympy pandas \
+six pyyaml pycollada lxml xlutils olefile requests blinker \
+opencv qt.py nine docutils calculix opencamlib ifcopenshell \
+appimage-updater-bridge"
```

Comment:
I recommend that you update to the latest script in the FreeCAD-Bundle repo as it has been adapted to work on aarch64 architecture too, it skips some packages that aren't available on that platform yet.